billchop-be/billchop-be/Controllers/LoansController.cs

```diff
@@ -0,0 +1,70 @@
+﻿using BillChopBE.DataAccessLayer.Filters;
+using BillChopBE.DataAccessLayer.Models;
+using BillChopBE.Services;
+using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace BillChopBE.Controllers
+{
+    [ApiController]
+    [Produces("application/json")]
+    [Route("api/loans")]
+    public class LoansController : ControllerBase
+    {
+        private readonly ILoanService loanService;
+
+        public LoansController(ILoanService loanService) 
+        {
+            this.loanService = loanService;
+        }
+
+        /// <summary>
+        /// Get all loans that need to be paid back to specific user. AKA the loans the user provided.
+        /// </summary>
+        /// <param name="loanerId">Id of user who lent money.</param>
+        /// <param name="groupId">Optional Id of context group.</param>
+        /// <returns></returns>
+        [HttpGet("provided-loans/{loanerId}")]
+        public async Task<ActionResult<IList<Loan>>> GetProvidedLoans(Guid loanerId, Guid? groupId)
+        {
+            return Ok(await loanService.GetProvidedLoans(loanerId, groupId));
+        }
+
+        /// <summary>
+        /// Get all loans that specific user has to pay back. AKA, the loans the user received.
```

Comment:
You could change this to something like "Get all loans of a specific user or, if group ID is specified, get all loans of specific user in a specific group."